public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -144,3 +189,17 @@ function inject_editor_template( $template ) {
 	return $template;
 }
 add_filter( 'template_include', __NAMESPACE__ . '\inject_editor_template' );
+
+/**
+ * Add a rewrite rule to handle editing a pattern.
+ */
+function rewrite_for_pattern_editing() {
+	add_rewrite_rule( '^pattern/(\d+)/edit', 'index.php?pagename=new-pattern&' . PATTERN_ID_VAR . '=$matches[1]', 'top' );
+
+	if ( isset( $_GET['post'] ) && isset( $_GET['action'] ) && 'edit' === $_GET['action'] && POST_TYPE !== get_post_type( $_GET['post'] ) && ! is_admin() ) {
```

Comment:
This would be easier to read on multiple lines, and I think there's a logic error here, should it be `POST_TYPE === get_post_type( $_GET['post'] )`?

```suggestion
	if ( 
		isset( $_GET['post'] ) &&
		isset( $_GET['action'] ) &&
		'edit' === $_GET['action'] &&
		POST_TYPE !== get_post_type( $_GET['post'] )
		&& ! is_admin()
	) {
```

Reply:
Meeep.. that was a test. 🙇

Reply:
By the way, I formatted it using `format:php`... it doesn't match the example above.

Reply:
Does it reformat the multiple-line version? The phpcs linter fix isn't as aggressive as prettier, so it's still up to the human developer to make code style decisions 🙂

Reply:
It does a multi-line format but the first`isset` is pulled up into the line as`if(`... etc...